dotcom-rendering/src/components/SignInGateSelector.importable.tsx

```diff
@@ -671,7 +713,13 @@ const SignInGateSelectorAuxia = ({
 		});
 	}, [abTest]);
 
-	if (isSignedIn || isUndefined(pageViewId)) {
+	// We are not showing the gate if we are in preview, it's a paid contents
+	// or the user is signed in or if for some reasons we could not determine the
+	// pageViewId
+
+	// According to the reacts rules we can only put this check after all the hooks.
+
+	if (isPreview || isPaidContent || isSignedIn || isUndefined(pageViewId)) {
```

Comment:
I guess technically these new checks are redundant because they're also in the `useOnce`, but doesn't hurt

Reply:
@tomrf1 
I am just expressing intent so that future readers know that 

1. The latter checks are the primary checks, they implement the primary policy

2. The hook checks have a different purpose, they only exist because Auxia is not stateless. Auxia count how many times we call them. If if wasn't for the Auxia being stateful, the checks in the hook are the ones that would be redundant.